README.md

```diff
@@ -8,6 +8,15 @@
 
 This module implements Event Streams with topics, partitions, throughput, storage size, cleanup policy, retention time, retention size, segment size and schema.
 
+## Note for KMS encryption
+
+The Event Streams service supports payload data encryption using a root key CRN of a Key Management Services like Key Protect or Hyper Protect Crypto Services (HPCS), see https://cloud.ibm.com/docs/EventStreams?topic=EventStreams-managing_encryption
+The root key CRN can be specified using the `kms_key_crn` variable.
+The authorization policy to allow the Event Streams service to access the key management service instance as a Reader MUST be configured in advance, see https://cloud.ibm.com/docs/account?topic=account-serviceauth
+This cannot be managed in the same Terraform state as the Event Streams service instance, because on destroy the instance is soft deleted to allow for recovery.
+The authorization policy must still be inplace when the instance is hard deleted or restored, otherwise the de-registration of the instance from the root key will fail on the backend, and you will not be able to delete the root key without creating a support case,
+for more details see https://cloud.ibm.com/docs/EventStreams?topic=EventStreams-managing_encryption#using_encryption
```

Comment:
```suggestion
## About KMS encryption

The Event Streams service supports payload data encryption that uses a root key CRN of a key management service, such as Key Protect or Hyper Protect Crypto Services (HPCS). You specify the root key CRN with the `kms_key_crn` input. For more information, see [Managing encryption in Event Streams](https://cloud.ibm.com/docs/EventStreams?topic=EventStreams-managing_encryption).

Before XXX, configure an authorization policy to allow the Event Streams service to access the key management service instance with the reader role. For more information, see [Using authorizations to grant access between services](https://cloud.ibm.com/docs/account?topic=account-serviceauth). 

You can't manage the policy in the same Terraform state file as the Event Streams service instance because the instance is soft deleted to support recovery of the key when you issue a `destroy` command. The authorization policy must exist when the instance is hard deleted or restored or the de-registration of the instance from the root key fails on the backend. If the policy doesn't exist, the only way to delete the root key is by opening a support case because. For more information, see [Using a customer-managed key](https://cloud.ibm.com/docs/EventStreams?topic=EventStreams-managing_encryption#using_encryption).
```
There's a lot of detail in here. Here's my first attempt.

**Questions**
first sentence: is it rather, "payload data encryption _by using_ a root key CRN..." 
`using` is ambiguous

2nd para: 
- XXX: In advance of what?
- You can't manage the **policy** or the **service** or something else in the Terraform?
`This` is ambiguous.

Reply:
> XXX: In advance of what?

^^ running the module

Reply:
> to support recovery of the key

its not recovery of the key, its recovery (reclamation) of the instance

Reply:
> the only way to delete the root key

^ the only way to re-register the instance from the root key, which is required if you want to delete it ....